lib/capistrano/gitflow/legacy/.LCKgitflow.rb~

```diff
@@ -0,0 +1 @@
+/home/raul/workspace/github/capistrano-gitflow/lib/capistrano/gitflow/recipes/gitflow.rb
```

Comment:
This looks like it should be removed. Might also want to do a global gitignore to prevent files like this from being accidentally added.